lib/lightning_web/endpoint.ex

```diff
@@ -16,7 +16,11 @@ defmodule LightningWeb.Endpoint do
     websocket: [connect_info: [session: @session_options]]
 
   socket "/worker", LightningWeb.WorkerSocket,
-    websocket: [error_handler: {LightningWeb.WorkerSocket, :handle_error, []}],
+    websocket: [
+      error_handler: {LightningWeb.WorkerSocket, :handle_error, []},
+      compress: true,
+      max_frame_size: 11_000_000
```

Comment:
I am wondering if the compression works before or after the frame size is validated.